CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [2.4.1](https://github.com/nuxt/nuxt.js/compare/v2.4.0...v2.4.1) (2019-01-29)
+
+
+### Bug Fixes
+
+* keepAliveProps broken in <nuxt-child> ([#4521](https://github.com/nuxt/nuxt.js/issues/4521)) ([b48b220](https://github.com/nuxt/nuxt.js/commit/b48b220))
+
```

Comment:
Not sure whether we want to completely remove the top-level changelog or correctly update it

Reply:
@manniL Lerna automatically does it because we forgot to update it.

Reply:
Yeah, I know. But as we use `release notes` that could confuse ppl.

Reply:
Reverted lerna version commit. We can gitignore top-level CHANGELOG.